faq.rst

```diff
@@ -80,3 +80,38 @@ The remedy is to disable the homepage in the pipelinesettings:
         }
         ]
       }
+
```

Comment:
Is the FAQ page the best page for it? I would expected it under the plugin section (and additionally referenced from the facade section)

Reply:
good point, did both.